src/components/Search/index.tsx

```diff
@@ -97,22 +93,40 @@ function Search({query, policyIDs, sortBy, sortOrder}: SearchProps) {
     const shouldShowEmptyState = !isLoadingItems && isEmptyObject(searchResults?.data);
 
     if (isLoadingItems) {
-        return <TableListItemSkeleton shouldAnimate />;
+        return (
+            <>
+                <SearchPageHeader
+                    selectedItems={{}}
```

Comment:
Could we just add a default to the component prop instead of passing this empty object? This just looks odd.